bin/improver-nbhood

```diff
@@ -72,8 +75,8 @@ def main():
         radius_or_radii_in_km = args.radii_in_km_by_lead_time[0].split(",")
         lead_times = args.radii_in_km_by_lead_time[1].split(",")
     result = (
-        BasicNeighbourhoodProcessing(
-            radius_or_radii_in_km,
+        NeighbourhoodProcessing(
+            args.kernel_method, radius_or_radii_in_km,
```

Comment:
No way to set unweighted mode option. Is the implementation and testing of this option technical_debt. Is this captured somewhere